.storybook/preview.js

```diff
@@ -43,4 +46,16 @@ export const parameters = {
       date: /Date$/,
     },
   },
-}
\ No newline at end of file
+  locale: reactIntl.defaultLocale,
+  locales: {
+    "en-US": { title: "English (US)", left: "🇺🇸" },
+    fr: { title: "Français", left: "🇫🇷" },
+    unknown: { title: "Unknown", left: "🚫" }
```

Comment:
Why do we have this? On safari at least, it renders strings from a random language (?) instead of the keys which is what I'd assume this to do

Reply:
- Some components are set to render English if the string resource file doesn't exist for the specified locale, but other components may just render something very basic or fallback on a message id.
- For `TripDetails`, the fare portion is no part of the language strings, it is set via config, so for the story, the behavior is not defined if the fare strings have not been set for the locale.

Reply:
This makes a lot of sense thanks!